### PR TITLE
add ability to select issues or prs for closing

### PR DESCRIPTION
**Pull request by kdaily**
*Description of changes:*

Allow configuration of closing issues, pull requests, or both. No tests were updated.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/entrypoint.js [hunk 1]
```diff
@@ -72,6 +73,19 @@ async function processIssues(client, args) {
     log.debug(`last updated ${issue.updated_at}`);
     const isPr = 'pull_request' in issue ? true : false;
 
+    if (isPr && args.issueTypes.indexOf(`pull_requests`) == -1) {
```
Comment (debora-ito) [inline src/entrypoint.js:76]: Can I suggest a different input name, maybe "processIssueTypes"?
This line would be less confusing.

File: test/local-docker.env [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # This environment file is used for local testing with Docker.
 
 # A personal access token with repo permissions
-REPO_TOKEN=
+REPO_TOKEN=548caf8b629b3dab533b72b7006eaff2eba87ea1
```
Comment (debora-ito) [inline test/local-docker.env:4]: Oops.

**Comment by debora-ito** [comment]
Can you add a description of the new field in [action.yml](https://github.com/aws-actions/stale-issue-cleanup/blob/master/)?

File: src/entrypoint.js (hunk 1 above)
Comment (kdaily) [reply src/entrypoint.js]: addressed this by adding consts with better names.

File: action.yml [hunk 3]
```diff
@@ -50,6 +53,7 @@ runs:
   image: 'Dockerfile'
   env:
     REPO_TOKEN: ${{ inputs.repo-token }}
+    ISSUE_TYPES: ${{ inputs.issue_types }}
```
Comment (debora-ito) [inline action.yml:56]: Typo: inputs.issue-types

**Comment by kellertk** [comment]
@kdaily please squash your commits. Also, you need to sign your commit before posting a PR. (See https://docs.github.com/en/github/authenticating-to-github/about-commit-signature-verification)

**Comment by kdaily** [comment]
@debora-ito the default I set is only applicable if run through the Action. I can set a default in the code as well, but thought that the code would never run outside of the Action, except for testing/dry run purposes with the provided `local-docker.env` file (which has `ISSUE_TYPES` set). Would you still prefer it to get a default there too (or, to fail more gracefully)?

**Comment by debora-ito** [comment]
@kdaily Got it. If it works through the Action run then no change is needed.

**Review by debora-ito — APPROVED** [review]

**Pull request merged**